app/services/analytics_events.rb

```diff
@@ -1459,6 +1459,20 @@ def idv_gpo_confirm_start_over_visited(**extra)
     track_event('IdV: gpo confirm start over visited', **extra)
   end
 
+  # The user ran out of time to complete their address verification by mail.
+  # @param [String] user_id UUID of the user who expired
+  # @param [Boolean] user_has_active_profile Whether the user current has an active profile
+  # @param [Integer] letters_sent Total # of GPO letters sent for this profile
+  def idv_gpo_expired(user_id:, user_has_active_profile:, letters_sent:, **extra)
```

Comment:
I think we want this event to be per profile and have the profile id too, and maybe `gpo_verification_pending_at` as discussed in the previous comment. Since you're not notifying in this PR, you don't have to do user-level queries that would tell you if they have an active profile. I would consider leaving that for a separate report if people want to know that info.

I would recommend discussing this analytics event with Ponja, because she has lots of ideas about info to track.

Reply:
I'd like to keep `user_has_active_profile`, since it might be interesting for reasons other than just notifications. Good point about reaching out to UX, I will do that.